src/test/scala/com/cognite/sdk/scala/v1/SessionsTest.scala

```diff
@@ -347,4 +347,32 @@ class SessionsTest extends SdkTestSpec with ReadBehaviours {
     val error = the[CdpApiException] thrownBy client.sessions.refresh(RefreshSessionRequest(123, "invalid-sessionKey"))
     error.message shouldBe "Session not found"
   }
+
+  it should "revoke a session" in {
+    val expectedIds = Seq(
+        CogniteInternalId(1),
+        CogniteInternalId(2)
```

Comment:
so invalid ids get "ok, revoked" response? Would it be possible to reveal existence of session by id via response code difference? :thinking: Not directly related to sdk though

Reply:
nvm, noticed here we mock responses, but the case may still be interesting